src/App.tsx

```diff
@@ -44,6 +44,16 @@ function App() {
     textarea
   );
   const setTextToParseThrottle = useThrottleCallback(setTextToParse, 2);
+  const inputRef = useRef({} as HTMLTextAreaElement);
+  const [currentPosition, setCurrentPosition] = useState({
+    selectionStart: textarea.length,
+    selectionEnd: textarea.length
+  });
+  React.useEffect(()=> {
+    inputRef.current.selectionStart = currentPosition.selectionStart;
+    inputRef.current.selectionEnd = currentPosition.selectionEnd;
+    // eslint-disable-next-line
+  }, [textarea]);
```

Comment:
The linter probably wants to have all of the dependencies included in this array, it's safer to add all of the dependencies rather than ignoring the linter. You can add inputRef as a dependency, and check that it is not the same as the current value rather than ignoring the linter warning